### PR TITLE
(SP: 1) Write unit test for "Find offers" page #97

### LINKED ISSUE #97: (SP: 1) Write unit test for "Find offers" page
### Page unit test
Unit test for "**Find offers**" page
Scenaries descriptions:
- [x] It should render page
- [x] Text of the page must be in the document

[Link to component](https://github.com/Space2Study-UA-1156/Client-02/blob/develop/src/pages/find-offers/FindOffers.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by docia**
Screenshot 2024-04-30 at 22 02 24

### CODE REVIEW

**Review by BodiaPutiak — APPROVED** [review]

**Review by ohanovdmytro — APPROVED** [review]

File: tests/unit/components/loader/Loader.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Loader from '~/components/loader/Loader.jsx'
```
Comment (Olenka-Hryk) [inline tests/unit/components/loader/Loader.spec.jsx]: These changes are already included in this PR #111. Could you please revert then or change the base branch from develop to #111 ?

Comment (docia) [reply tests/unit/components/loader/Loader.spec.jsx]: thanks. fixed!

**Review by Olenka-Hryk — APPROVED** [review]

File: tests/unit/pages/find-offers/FindOffers.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: Mock the `PageWrapper` component. Also, both tests are the same)

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/pages/find-offers/FindOffers.spec.jsx
Comment (docia) [reply tests/unit/pages/find-offers/FindOffers.spec.jsx:1]: mocked PageWrapper and edited one test

File: tests/unit/pages/find-offers/FindOffers.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          <div data-testid='page-wrapper'>Mocked PageWrapper</div>
+        ))
+      }
+    })
+    render(<FindOffers />)
+  })
+
+  it('renders the page', () => {
+    render(<FindOffers />)
+  })
```
Comment (vlad-khrychov) [inline tests/unit/pages/find-offers/FindOffers.spec.jsx]: Look to the sonar messages.
image
I think `element= screen.getByTestId('page-wrapper')` would be great.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**